provider-ci/internal/pkg/config.go

```diff
@@ -315,6 +315,14 @@ type Config struct {
 	// AllowMissingDocs controls whether we allow missing docs in the provider.
 	// Defaults to false.
 	AllowMissingDocs bool `yaml:"allowMissingDocs"`
+
+	// NoSchema is useful for providers such as parameterized providers that do not check in
```

Comment:
```suggestion
	// NoSchema is set by providers such as parameterized providers that do not check in
```

It seems like setting this is a necessity not just useful.